types/lru-cache/index.d.ts

```diff
@@ -1,23 +1,127 @@
-// Type definitions for lru-cache 4.1
+// Type definitions for lru-cache 5.0
 // Project: https://github.com/isaacs/node-lru-cache
 // Definitions by: Bart van der Schoor <https://github.com/Bartvds>
 //                 BendingBender <https://github.com/BendingBender>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.3
 
-export = LRU;
-
-declare const LRU: LRU;
-
-interface LRU {
-    <K, V>(opts?: LRU.Options<K, V>): LRU.Cache<K, V>;
-    <K, V>(max: number): LRU.Cache<K, V>;
-    new <K, V>(opts?: LRU.Options<K, V>): LRU.Cache<K, V>;
-    new <K, V>(max: number): LRU.Cache<K, V>;
+declare class LRUCache<K = any, V = any> {
```

Comment:
I wouldn't do the `= any` dance here any longer. It was necessary to prevent breaking users' code without a major version bump which is exactly what is being done here.

Reply:
Ok, I will make both template parameters are required.